Changelog.md

```diff
@@ -4,10 +4,6 @@ This is the changelog, summarising changes in each version
 
 Minor changes may be ommited, as well as improvements to documentation.
 
-## Unreleased
-
-Bumped MSRV to 1.63 (`libc` crate requires it).
```

Comment:
yes, I forgot actually change the version in `Cargo.toml` and only wrote it in the changelog